plugins/woocommerce-blocks/assets/js/blocks/cart/block.js

```diff
@@ -26,11 +26,19 @@ import './style.scss';
 const reloadPage = () => void window.location.reload( true );
 
 const Cart = ( { children, attributes = {} } ) => {
-	const { cartIsLoading } = useStoreCart();
+	const cart = useStoreCart();
 	const { hasDarkControls } = attributes;
 
+	const { dispatchStoreEvent } = useStoreEvents();
+
+	// Ignore changes to dispatchStoreEvent callback so this is ran on first mount only.
+	useEffect( () => {
+		dispatchStoreEvent( 'cart-render', { cart } );
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [] );
```

Comment:
This should allow you to remove the `// eslint-disable-next-line react-hooks/exhaustive-deps` and not need to pass `cart`.

Reply:
It seems to me we still need the empty array and the eslint diable comment to ensure the event is fired on first mount only, just like for the `render-checkout-form`.

Reply:
I will test this again, but if you don't pass cart, there shouldn't be any comments, the `[]` is always needed yeah.

Reply:
Okay so it's `dispatchStoreEvent` and `dispatchCheckoutEvent` that aren't stable, this is fine for now but in the future we will change this so a better event is emitted from Checkout.